metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorFactory.java

```diff
@@ -106,15 +114,17 @@ private static HiveConf getDefaultConf() {
         result.setInt("javax.jdo.option.DatastoreWriteTimeoutMillis", 60000);
```

Comment:
I'd make all these strings constants with names as well as their default values you're setting here. That way we can quickly change them in one place if we have to use them in multiple places.